src/ios-deploy/ios-deploy.m

```diff
@@ -596,11 +596,11 @@ void mount_developer_image(AMDeviceRef device) {
     NSLogVerbose(@"Developer disk image: %@", image_path);
 
     FILE* sig = fopen(CFStringGetCStringPtr(sig_path, kCFStringEncodingMacRoman), "rb");
-    void *sig_buf = malloc(128);
-    size_t bytes_read = fread(sig_buf, 1, 128, sig);
-    assert( bytes_read == 128);
+    size_t buf_size = 128;
+    void *sig_buf = malloc(buf_size);
+    assert(fread(sig_buf, 1, buf_size, sig) == buf_size);
```

Comment:
I'm pretty sure this fread() shouldn't be inside the assert

Reply:
Yep, fixed.